css/tablet-style.css

```diff
@@ -0,0 +1,296 @@
+/* Mobile Style Sheet */
+
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
```

Comment:
The other benefit of combining everything into 1 stylesheet is you don't have to replicate styles like this into 2 files